lecture-notes/2-vcs.md

```diff
@@ -519,16 +519,25 @@ single unit and so they might as well have just been one to begin with.
     Ensuring each commit stands alone is also known as "keeping the `main`
     branch green," and we'll talk about it more in the final module.
 
+Conversely, if you get in the habit of bundling multiple independent changes into a single commit, Git also becomes less useful. Subcommands such as `git revert`---which undoes a commit's changes---work best when commits are as granular as possible. In fact, Git makes it almost impossible to work with historical changes more granular than a commit.
```

Comment:
idk what this means about more granular than a commit being impossible. partial reverts/adds/etc are a thing i use regularly

Reply:
git is commit-addressable like memory is byte-addressable

fundamental unit

etc